app/actions/wallets-actions.js

```diff
@@ -4,7 +4,7 @@ import Action from './lib/Action';
 // ======= WALLET ACTIONS =======
 
 export default class WalletsActions {
-  createWallet: Action<{ name: string, currency: string }> = new Action();
+  createWallet: Action<{ name: string, currency: string, password: string }> = new Action();
```

Comment:
Yeah but that's only an implementation detail inside the form … I think you should not submit an empty string password at all (we should only allow valid data to be sent). So i would prefer it to be `null` in case of no password is set